docs/build/html/_sources/creating-installers.rst.txt

```diff
@@ -0,0 +1,83 @@
+===========================================
```

Comment:
are the documents inside of `docs/build` generated by sphinx? if so, they should be removed from the repo.